tests/eval_files/async232_asyncio.py

```diff
@@ -0,0 +1,260 @@
+# type: ignore
+# NOTRIO # see async232.py
+# NOANYIO # see async232.py
+import io
+from io import BufferedRandom, BufferedReader, BufferedWriter, TextIOWrapper
+from typing import Any, Optional
+
+blah: Any = None
+
+
+async def file_text(f: io.TextIOWrapper):
+    f.read()  # ASYNC232_asyncio: 4, 'read', 'f'
+    f.readlines()  # ASYNC232_asyncio: 4, 'readlines', 'f'
+
+    # there might be non-sync calls on TextIOWrappers? - but it will currently trigger
+    # on all calls
+    f.anything()  # ASYNC232_asyncio: 4, 'anything', 'f'
+
+
+async def file_binary_read(f: io.BufferedReader):
+    f.read()  # ASYNC232_asyncio: 4, 'read', 'f'
+
+
+async def file_binary_write(f: io.BufferedWriter):
+    f.read()  # ASYNC232_asyncio: 4, 'read', 'f'
```

Comment:
`f.write()`?  And below for the bare name.

Reply:
This is tested by row 16/17, *any* call on a file object will trigger the error. Added clarifying comment and a more obvious nonsense-name example.

(pretty much all your comments in this vein make me go "yeah duh", change it, then after a minute realize/remember the reason for the status quo :sweat_smile:)